### DISCUSSION TITLE
How to make logging work?

### DISCUSSION THREAD

**Discussion by DomonkosSuranyi** [bevyengine/bevy#2852] [Q&A] [answered]
Hi there,
I'm having troubles with getting the log work.
It's on server side so I don't need DeafultPlugins just MinimalPlugins.
Here's the snippet
```rust
fn main() {
    App::build()
        .insert_resource( LogSettings { level: Level::INFO, filter: "".to_string() })
        .add_plugin(LogPlugin::default())
        .add_plugins(MinimalPlugins)
        .add_system(logging_something_nice_every_frame.system())
        .run();
}
```

The program is running but no logs shown on the console.
What am I doing wrong?

Thx for help

**Comment by Nilirad** [comment]
You can check out the examples.

If you're targeting the main branch of Bevy:
https://github.com/bevyengine/bevy/blob/main/examples/app/logs.rs

If you are targeting the latest Bevy release:
https://github.com/bevyengine/bevy/blob/latest/examples/app/logs.rs

**Reply by DomonkosSuranyi** [reply]
Thanks, but my trials were based on this example.
I hope it works for others. I've already decided to use `env_logger`, it works fine.

**Comment by jakobhellermann** [comment, marked as answer]
In `logging_something_nice_every_frame`, are you logging using `bevy_log::info!` (a reexport of `tracing::info`) or using `log::info` from the `log` crate? The latter doesn't automatically get picked up by `LogPlugin`s tracing subscriber.

**Reply by jakobhellermann** [reply]
If you have a dependency that uses `log` and want to log their logs from inside bevy, you can use https://docs.rs/tracing-log/0.1.2/tracing_log/

**Reply by DomonkosSuranyi** [reply]
cool. It works well with bevy_log::info!
Thx
